Assets/src/QuestionStore/QuestionStore.cs

```diff
@@ -1,17 +1,61 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 using System.Linq;
 
 public class QuestionStore : MonoBehaviour
 {   
+	public Text sector, sector1, sector2, sector3, sector4, sector5, sector6, sector7, sector8, sector9, sector10, sector11; 
```

Comment:
If this is for the UI it should probably be in its own class

Reply:
You want me to create a new script that only deals with setting the categories on the wheel and place it in the src/QuestionStore dir?

Reply:
well it would be the Wheel class in the UI but we can move things later if you want to just finish up this review

Reply:
sounds good to me